nabairqualityd/aqicn.py

```diff
@@ -35,15 +38,17 @@ def _aqicn_url(self, lat, lon):
         if lat and lon:
             # Use geolocalized API
             return (
-                "https://api.waqi.info/feed/geo:" + lat + ";" + lon + "/"
-                "?token=4cf7f445134f3fb69a4c3f0e5001e507a6cc386f"
+                AQICN_FEED
+                + "geo:"
+                + lat
+                + ";"
+                + lon
+                + "/?token="
+                + AQICN_TOKEN
```

Comment:
I'm not a python expert, but using string substitution is maybe better:
`return f"{AQICN_FEED}geo:{lat};{lon}/?token={AQICN_TOKEN}"

Reply:
Any tips @pguyot?

Reply:
Indeed f-strings are considered a better practice for Python 3.6+ unless format string is user-supplied.

Reply:
Hmmm.... Existing Pynab code makes extensive use of string catenation. I didn't want to disrupt this.
I suggest you conduct an extensive code review/cleanup for 1.0.0.